src/abstractdataframe/abstractdataframe.jl

```diff
@@ -518,27 +526,28 @@ function Base.convert{T}(::Type{Matrix{T}}, df::AbstractDataFrame)
     n, p = size(df)
     res = Array(T, n, p)
     idx = 1
-    for col in columns(df)
-        anyna(col) && error("DataFrame contains NAs")
-        copy!(res, idx, data(col))
+    for (name, col) in zip(names(df), columns(df))
+        anynull(col) && error("cannot convert a DataFrame containing null values to array (found for column $name)")
```

Comment:
Not specifically related to this PR, but this error message may give the (possibly false) impression that only the column mentioned in the error has null values. Sorry if this is pedantic.